src/guides/v2.3/config-guide/prod/config-reference-systemxml.md

```diff
@@ -249,6 +249,22 @@ A `<field>`-Tag can have the following values for the `type=""` attribute:
 | `time`          | Control to set time using three dropdowns–Hour, minute and second.                                                                                                                                         |
 | `allowspecific` | A multiselect list of specific countries. Requires a `source_model` such as `Magento\Shipping\Model\Config\Source\Allspecificcountries`                                                                               |
 | `image`         | Allows an image to be uploaded.                                                                                                                                                                                                                                                                                                                                   |
+| `note`          | Allows an informational note to be added to the page. This type requires a `frontend_model` to render the note. |
+
+It is also possible to create a custom field type. This is often done when a special button, with an action, is required. To do this requires two main elements:
+
+-  Creating a Block in the `adminhtml` area
+-  Setting the `type="""` to the path to this block
+
+The block itself requires, at a minimum, a `__contstuct` method and a `getElementalHtml()` method. The [Magento_OfflineShipping](https://github.com/magento/magento2/tree/2.4-develop/app/code/Magento/OfflineShipping) is a simple example of a custom type.
```

Comment:
`__construct`
The github link will fail after 2.4-develop is merged. There is a var for the repo.